src/reader/LikeFeedbackBox.js

```diff
@@ -18,9 +18,9 @@ export default function LikeFeedbackBox({
 
   return (
     <s.FeedbackBox>
-      <small>{strings.helpUsMsg}</small>
+      {/*<h4>{strings.helpUsMsg}</h4>*/}
```

Comment:
Proposal: We think this is too much of a prompt to read for the user.

Reply:
I like the short message! 

I would beta-test it with users maybe? Because the semantics of 
- Did you enjoy the article?, and
- Would you like more articles like this?

are different. 

What I'm not sure about the second is whether:
- the user will understand what is meant by "this"
- asking me whether i liked something is an easier question - because i know if i liked it or not. asking me whether i want more is a more difficult question, e.g. i might have liked it, but not want more.

Reply:
Good points about the semantics of our proposal. 
It is true that it might be a bit ambiguous for the user to interpret what "this" is in the context of the article. As it could be the topic, the difficulty level or something else.

It is a more difficult choice to make. We thought that the user might be a bit more incentivized to click on something that would have an effect on their experience. But we agree that, it is better to keep it simple.

We will revert the changes back to the initial message 😊

Reply:
Original was too long. New was ambiguous. Maybe there could be one that's concise and unambiguous? 

Hegel: thesis, antithesis => synthesis :)